mmcif/io/BinaryCifWriter.py

```diff
@@ -148,9 +153,30 @@ def __getAttributeType(self, dObj, atName):
                 logger.error("Undefined type for category %s attribute %s - Will treat as string", dObj.getName(), atName)
```

Comment:
Can this be made a `warning` instead of an `error`? (An oversight by me in my previous PR)

Reply:
Also note that this occurs in line 128 of `DataCategoryTyped.py` as well.

Reply:
I did not change this - I do not think.  We can change to warning.  I think when you implemented not pure dictionary based - this should have been changed... Will do.

Reply:
Yes sorry about that, this was an oversight from my earlier PR. Thank you for making the change here.

Reply:
See new commit